config.yaml.sample

```diff
@@ -210,6 +217,10 @@ toi-prune:
     path: osm
     layer: all
     format: zip
+  # a reduced version of prev s3 entity
+  store:
+    layer: all
+    format: zip
```

Comment:
My thoughts about the configuration:

* we can probably just use `store` throughout and grow any api needed to support any operations across all backends.
* regarding the duplication, on the one hand it's nice to just have it in one place. But in practice, it's very useful to have the configuration for the different operations separate, which allows us to handle production deploys and moments of transition more readily. Yaml allows us to specify pointers, so maybe we can do something clever here to refer to other sections?

What do you think @iandees ?